packages/apollo/package.json

```diff
@@ -57,15 +58,19 @@
     "graphql": "14.0.2 - 14.2.0 || ^14.3.1",
     "graphql-tag": "2.10.1",
     "heroku-cli-util": "8.0.11",
+    "line-column": "1.0.2",
     "listr": "0.14.3",
     "lodash.identity": "3.0.0",
     "lodash.pickby": "4.6.0",
     "moment": "2.24.0",
+    "prettier": "1.18.2",
+    "recast": "0.18.1",
     "strip-ansi": "5.2.0",
     "tty": "1.0.1",
     "vscode-uri": "1.0.6"
   },
   "devDependencies": {
+    "@types/prettier": "^1.18.0",
```

Comment:
IIRC, `typescript` is the exception to the rule (though I don't remember why). All other `devDependencies` should be declared in the monorepo's root `package.json`.

Reply:
moved it